docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+services:
+  arma-reforger:
+    build: .
+    image: ghcr.io/acemod/arma-reforger:edge
+    platform: linux/amd64
+    container_name: arma-reforger
+    network_mode: host
+    volumes:
+        - /path/to/configs:/reforger/Configs
+        - /path/to/profile:/home/profile
```

Comment:
I'd change this to a directory in the current working directory or switch to using volumes.

That allows someone to copy past and run it right away. Advanced users can modify regardless.

Reply:
Working directory, full volumes don't really make sense here.